dotCMS/src/main/java/com/dotmarketing/common/reindex/ReindexQueueAPIImpl.java

```diff
@@ -219,14 +222,14 @@ public void deleteReindexEntry(String identiferToDelete) throws DotDataException
         reindexQueueFactory.deleteReindexEntry(identiferToDelete);
     }
 
-
-
     @Override
     @WrapInTransaction
     public void markAsFailed(final ReindexEntry idx, final String cause) throws DotDataException {
-        Logger.warn(this.getClass(), "Reindex failed for :" + idx + " because " + cause);
         reindexQueueFactory.markAsFailed(idx, UtilMethods.shortenString(cause, 300));
 
+        new Thread(
```

Comment:
Not sure if we should create thread, DotConcurrentFactory is your friend

Reply:
done https://github.com/dotCMS/core/pull/18604/commits/fc979e0e1d04475951dbb213b379b17022474a8e#diff-25af5da06ceca449aa6c6a84fca39091R231